src/Interfaces/Interfaces.tsx

```diff
@@ -0,0 +1,11 @@
+export interface IPerson {
+  [key: string]: any;
```

Comment:
if you declare all of the fields below, you don't need this. remove

Reply:
done